0001-template.md

```diff
@@ -1,12 +1,12 @@
 ## Summary
 [summary]: #summary
 
-One paragraph explanation of the feature.
+One paragraph explanation of the feature. Describe the "why" and feature benefit.
 
 ## Motivation
 [motivation]: #motivation
 
-Reflect the will of the PRD (Product Requirements Document). If a linkable artifact, include it here. If not, summarize in your own words.
+Reflect the will of the PRD (product requirements document). If the PRD is a linkable artifact, include the link here. If not, summarize the product requirement in your own words. Describe the delta of this change.
```

Comment:
Kudos to @mrmr1993  who provided a theory overview in our first RFC! https://github.com/o1-labs/rfcs/pull/2/files#diff-2c04d1657aa6e49b23d97667412e570417f73bc72b2f5b32dca24196832504c8R18 

should we can add suggestions here in the template to encourage this level of context?